package.json

```diff
@@ -15,36 +22,71 @@
     "@types/node": "^16.11.39",
     "@types/react": "^18.0.12",
     "@types/react-dom": "^18.0.5",
+    "@types/react-redux": "^7.1.25",
+    "axios": "^1.3.5",
+    "bootstrap": "^5.2.3",
     "classnames": "^2.3.1",
-    "eslint-plugin-jest": "^27.0.1",
+    "eslint-plugin-jest": "^27.2.1",
     "formik": "^2.2.9",
-    "material-react-table": "^0.40.8",
+    "material-react-table": "^1.11.3",
+    "primeicons": "^6.0.1",
+    "primereact": "^9.4.0",
+    "ramda": "^0.29.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
     "react-helmet-async": "^1.3.0",
-    "react-hook-form": "^7.34.2",
+    "react-hook-form": "^7.43.9",
+    "react-icons": "^4.8.0",
     "react-redux": "^8.0.2",
-    "react-router-dom": "^6.3.0",
-    "react-scripts": "5.0.1",
+    "react-router-dom": "^6.10.0",
+    "@craco/craco": "5.0.1",
     "redux": "^4.2.0",
     "redux-thunk": "^2.4.1",
     "sass": "^1.52.3",
-    "typescript": "^4.7.3",
+    "typescript": "^4.9.5",
+    "typescript-plugin-css-modules": "^5.0.1",
     "web-vitals": "^2.1.4",
     "yup": "^0.32.11"
   },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+  "devDependencies": {
+    "@babel/preset-typescript": "^7.18.6",
+    "@craco/craco": "^7.1.0",
+    "@types/ramda": "^0.29.0",
+    "@typescript-eslint/eslint-plugin": "^5.48.2",
+    "@typescript-eslint/parser": "^5.48.2",
+    "eslint": "8.22.0",
```

Comment:
`eslint-plugin-react-hooks` is missing from here, and my IDE complains, because the .eslintrc asks to load the `react-hooks` plugin.

A related comment, please ensure that no linting (including prettier) depends on an IDE-specific plugin.